benjaminkott/bootstrap_package
# [TASK] Remove deprecated TCEforms flexform XML element

# Pull Request

## Related Issues

* Fixes # [bug]

## Prerequisites

* [X] Changes have been tested on PHP 8.0
* [X] Changes have been tested on PHP 8.2

## Description

Using TCEforms within flexform definitions has been deprecated in v12 and removed in v13:

https://docs.typo3.org/c/typo3/cms-core/main/en-us/Changelog/12.0/Deprecation-97126-TCEformsRemovedInFlexForm.html

This patch is compatible to v12 and v13 by removing this element.

(Otherwise, in v13, all flexform options would not be editable)

## Steps to Validate

1. In v13, create an carousel item. 
2. Check the flexform options in Tab "Carousel Options".
3. See that all option keys are listed, but no labels and inputs
